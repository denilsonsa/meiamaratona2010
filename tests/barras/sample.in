3
10101101110110001010111100010110100011011110101010101110011001101011100111001010010001110100101
10101101110110001010111100010110100011011110100010111110011001101111100111001000010001110100101
10100101110001001010011100111010110011001110101010101111011000101101000111101010001101110110101
