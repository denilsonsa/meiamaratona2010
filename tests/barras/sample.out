856943414089
XXXXXXXXXXXX
856943414089
